build.sh

```diff
@@ -9,7 +9,9 @@ mono ./.nuget/NuGet.exe restore ./ScriptCs.sln
 
 # script
 mkdir -p artifacts/Release/bin
-xbuild ./ScriptCs.sln /property:Configuration=Release /nologo /verbosity:normal
-cp src/*/bin/Release/* artifacts/Release/bin/
-mono ./packages/xunit.runners.1.9.2/tools/xunit.console.clr4.exe test/ScriptCs.Tests.Acceptance/bin/Release/ScriptCs.Tests.Acceptance.dll /xml artifacts/ScriptCs.Tests.Acceptance.dll.TestResult.xml /html artifacts/ScriptCs.Tests.Acceptance.dll.TestResult.html
+xbuild ./build/Build.proj /property:Configuration=Release /nologo /v:m /target:UpdateProjectVersions
+xbuild ./ScriptCs.sln /property:Configuration=Release /nologo /v:m /fl "/flp:LogFile=artifacts/msbuild.log;Verbosity=diagnostic;DetailedSummary"
+xbuild ./build/Build.proj /property:Configuration=Release /nologo /v:m /target:RestoreCommonVersionInfo
```

Comment:
We have to use `verbosity:normal` (`v:n`) with xbuild. See https://github.com/scriptcs/scriptcs/pull/739

> xbuild verbosity is set to normal rather than minimal because the minimal verbosity is useless when there are problems, e.g. you get error messages but it doesn't even tell you in which project the errors occur